packages/js/examples/react-audio/stories/components/DialPad.jsx

```diff
@@ -138,17 +165,18 @@ const DialPad = ({
 
 DialPad.propTypes = {
   call: PropTypes.object,
-  onDigit:PropTypes.func.isRequired,
+  onDigit: PropTypes.func.isRequired,
   onBackspace: PropTypes.func.isRequired,
   onStartCall: PropTypes.func.isRequired,
   onEndCall: PropTypes.func.isRequired,
   toggleHold: PropTypes.func.isRequired,
   toggleMute: PropTypes.func.isRequired,
+  toggleDeaf: PropTypes.func.isRequired,
   isHold: PropTypes.bool.isRequired,
-  isMute:PropTypes.bool.isRequired,
+  isMute: PropTypes.bool.isRequired,
+  isToggle: PropTypes.bool.isRequired,
```

Comment:
Is this supposed to be `isDeaf`? Otherwise I'm unsure what's being toggled by this property